internal/controller/cache/cache.go

```diff
@@ -0,0 +1,36 @@
+package cache
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	autoscalingv1 "k8s.io/api/autoscaling/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/cache"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/kyma-project/eventing-manager/internal/label"
+)
+
+func New(config *rest.Config, options cache.Options) (cache.Cache, error) {
+	return cache.New(config, applySelectors(options))
+}
+
+// applySelectors applies label selectors to runtime objects created by the EventingManager.
+func applySelectors(options cache.Options) cache.Options {
+	// TODO(marcobebway) filter by label "app.kubernetes.io/created-by=eventing-manager" when it is released
```

Comment:
the pr introducing this feature is already merged: https://github.com/kyma-project/eventing-manager/pull/264